scylla/src/transport/iterator.rs

```diff
@@ -141,16 +143,19 @@ impl RowIterator {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I think we should listen to clippy where possible - there are too many arguments here.

We could create a struct with all the parameters and pass it here
```rust
pub(crate) struct PreparedIteratorConfig {
    prepared: PreparedStatement,
    values: SerializedValues,
    ....
}
```

Reply:
Btw `PreparedStatement` is created by `Session`, so it would be possible to keep the determined consistency inside it. Although that would require much more changes, might not be worth it.

Reply:
I made al the changes expect for this one, not sure what you mean with it